src/doc/en/installation/conda.rst

```diff
@@ -152,12 +155,12 @@ Here we assume that you are using a git checkout.
   - Install the build prerequisites and the Sage library::
 
       $ pip install --no-build-isolation -v -v --editable ./pkgs/sage-conf ./pkgs/sage-setup
-      $ pip install --no-build-isolation -v -v --editable ./src
+      $ pip install --no-build-isolation --config-settings editable_mode=compat -v -v --editable ./src
```

Comment:
Why is this change needed?

Reply:
Because in modern editable installs (PEP 660 instead of `setup.py develop`), compile-time dependencies do not work well according to my tests: Every Cython file seems to be rebuilt.
The change here restores the behavior of setuptools that we used to get from the pinned setuptools.

Reply:
Thanks for the clarification. Maybe this should be documented somewhere?

Reply:
Sure, I'll add a quick note.

Reply:
Done in ec5ee1b, please take a look

Reply:
Thanks, looks good to me!